### PR TITLE
Task1: Add CLI

### PR DESCRIPTION
**Pull request by susankoju**
image (10)

### CODE REVIEW

File: Python/susankoju/task1/store_result.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        
+if __name__ == '__main__':
+    
+    parser = argparse.ArgumentParser(description='Store user information')
+    
+    parser.add_argument('--name', type=str, help='a name')
+    parser.add_argument('--dob', help='a date of birth(day/month/year)', type=dob_parser)
+    parser.add_argument('--score', type=float, help='the score in the subject')
+    parser.add_argument('--total', type=int, help='the total score in subject')
+    parser.add_argument('--subject', type=str, help='a name of subject')
```
Comment (sanjaymhj) [inline Python/susankoju/task1/store_result.py]: Please confirm to requirement: There should me limited choice for subject

**Review by sanjaymhj — CHANGES_REQUESTED** [review]

File: Python/susankoju/task1/store_result.py (hunk 1 above)
Comment (susankoju) [reply Python/susankoju/task1/store_result.py]: Okay dai

File: Python/susankoju/task1/store_result.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                data  = row.split(',')
+                if data[0].lower() == args.name.lower() and data[1] == str(args.dob):
+                    new = parse_data(args)
+                    new_data = new.split(',')
+                    for key in range(len(new_data)):
+                        try:
+                            if float(new_data[key]) < 1:
+                                new_data[key] = data[key]
+                        except Exception as e:
+                            pass
```
Comment (Basanta78) [inline Python/susankoju/task1/store_result.py]: raise or handle the expections

File: Python/susankoju/task1/store_result.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                    file_data =  ''.join(old_data).replace(row , ','.join(new_data))
+    except Exception as e:
+        #print(str(e))
+        pass
```
Comment (Basanta78) [inline Python/susankoju/task1/store_result.py]: .

**Pull request merged**